src/schema/v2/sale/index.ts

```diff
@@ -234,6 +237,62 @@ export const SaleType = new GraphQLObjectType<any, ResolverContext>({
           }
         },
       },
+      auctionsDetailFormattedStartDateTime: {
+        type: GraphQLString,
+        description:
+          "A more granular formatted description of when the auction starts or ends or if it has ended",
+        resolve: ({ start_at, end_at, ended_at }, _options) => {
+          return auctionsDetailFormattedStartDateTime(
+            start_at,
+            end_at,
+            ended_at
+          )
+        },
+      },
+      auctionsDetailFormattedStartEndSignal: {
+        type: GraphQLString,
+        description:
+          "A colored signal to users indicating how much time until sale opens/closes",
+        resolve: (
+          { start_at, end_at, ended_at },
+          _options,
+          { defaultTimezone }
+        ) => {
+          return auctionsDetailFormattedStartEndSignal(
+            start_at,
+            end_at,
+            ended_at,
+            defaultTimezone
+          )
+        },
+      },
+      auctionsDetailStartEndSignalColor: {
```

Comment:
I'm cool w/ getting this in ASAP and following up with a refactor but could be worth creating a new type `AuctionDetails` so the query would have more structured data:
```graphql
auctionDetails {
    formattedStartDateTime 
    formattedStartEndSignal {
        copy
        color
    }
    cascadingIntervalLabel
}
```

Reply:
Yea, this seems like a better structure to me for sure (vs. all root fields on `Sale`).

Reply:
yeah should definitely do that!